src/Components/ImportTable/ImportTable.tsx

```diff
@@ -107,8 +116,54 @@ export default function ImportTable(this: any) {
   ]);
 
   useEffect(() => {
-    getRows().then();
+    async function doSearch(cancelToken: CancelTokenSource) {
+      const checkedValues = [];
+      for (let i = 0; i < state.currentPerPage.value; i++) {
+        checkedValues.push(false);
+      }
```

Comment:
Har allergi for forløkker med index. Går ikke dette an å gjøre via 
`const checkedValues = new Array(state.currentPerPage.value).fill(false)`?

Reply:
Det er så mye jeg hadde lyst å endre, men måtte stoppe. `fill()` var faktisk nytt for meg. tnx